### ISSUE TITLE
NTR: "organism food management process"

### ISSUE THREAD

**Issue by ddooley** [FoodOntology/joint-food-ontology-wg#8]
Looking for a parent term that digestion process and fasting process could be parts of, and work in tandem.  It would involve an organism's whole food maintenance process - including the processes that give rise to hunger, or other food ingestion triggers, as well as mastication, digestion etc. 

label: food management process 
definition: The process an organism has which achieves a dietary pattern and involves food location, selection and episodic consumption.

**Comment by maweber-bia** [comment]
Think it is great to have this general parent term to group all the physiological  processes  linked to nutrition, including the neuro-regulation process.
But I don't catch the meaning of "food location" in the definition.  The definition should specify that the scope involves the whole food maintenance process as stated in your comment @ddooley

**Comment by ddooley** [comment]
For food location - I meant the location of food, i.e. availability - having to source it and obtain it.  This was to link to the concept of "food deserts" that I've seen as a research topic.  People may want nutritious food but its actually hard to obtain, leading to negative health consequences.

**Comment by FrancescoVit** [comment]
> For food location - I meant the location of food, i.e. availability - having to source it and obtain it. This was to link to the concept of "food deserts" that I've seen as a research topic. People may want nutritious food but its actually hard to obtain, leading to negative health consequences.

Really nice concept. If "location" is not clear, maybe going more directly to the point as you described with simply the wording "food availability"?

> label: food management process
> definition: The process an organism has which achieves a dietary pattern and involves food location, selection and episodic consumption.

If personal preferences, personal tastes, daily routine linked to choices in food (i.e. workload and eating a fresh-cut/convenience food; hoping that I picked up the correct translation), or similar concepts fall in this category, I would say that it could be the reverse. The "food management process" has as a result the dietary pattern. The ONS "dietary pattern" is actually am information content entity, so it should be right to have a process with this entity as output. 

In other words, during "food management process" choices about food happen, depending on multiple reasons, which are then concretized in a "dietary pattern", ultimately determining the "diet" (or correlated classes).

**Comment by ddooley** [comment]
Yes, I agree with all you've said!

**Comment by ddooley** [comment]
So
definition: The process an organism has which achieves a dietary pattern and involves food availability, physiological or psychological motivation, selection, episodic consumption, and digestion.

**Comment by maweber-bia** [comment]
agreed!

**Comment by laurenechan** [comment]
@ddooley I think I am a little unclear as to why digestion is involved in this term. Are you implying the physical capacity to consume and digest particular food types impacting food selection etc?

**Comment by ddooley** [comment]
Yes.  Say digestion process was faulty/diseased, then an organism's food management process might involve changing diet, or consuming more, etc.

**Comment by ddooley** [comment]
@FrancescoVit is this term finalized in ONS now?

**Comment by FrancescoVit** [comment]
No, lost this discussion. I'll self assign and open issue in ONS repo

**Comment by ddooley** [comment]
Completed March 21 2021!
